source/xconf-client/xconfclient.h

```diff
@@ -32,12 +32,9 @@
 #define TR181_DEVICE_MFR                          "Device.DeviceInfo.Manufacturer"
 #endif
 
-#if defined(WHOAMI_ENABLED)
-#define TR181_DEVICE_PARTNER_ID                     "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Bootstrap.PartnerName"
+#define TR181_DEVICE_PARTNER_NAME                   "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Bootstrap.PartnerName"
```

Comment:
Where is the corresponding change where we read this parameter instead of TR181_DEVICE_PARTNER_ID when whoami is enabled?